<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialportSAKIODeviceControler</class>
 <widget class="QWidget" name="SerialportSAKIODeviceControler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>162</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>串口控制面板</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QComboBox" name="comboBoxBaudrate"/>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>串口设备选择</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>数据位</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="comboBoxDatabits">
     <property name="currentText">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>波特率</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="comboBoxParity"/>
   </item>
   <item row="5" column="0">
    <widget class="QComboBox" name="comboBoxStopbits"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>停止位</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>校验位</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QComboBox" name="comboBoxSerialports"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
